h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -27,7 +27,7 @@
  * <p>
  * TODO: make this robust against removal of models from the DKV.
  */
-public class Leaderboard extends Keyed<Leaderboard> {
+public class Leaderboard extends Lockable<Leaderboard> {
   /**
```

Comment:
changes in Leaderboard have been back-ported to https://github.com/h2oai/h2o-3/pull/3898
As the refactoring raised this issue much more consistently.